pkg/controllers/nodeclaim/garbagecollection/controller.go

```diff
@@ -107,10 +107,9 @@ func (c *Controller) Reconcile(ctx context.Context) (reconcile.Result, error) {
 			"provider-id", nodeClaims[i].Status.ProviderID,
 			"nodepool", nodeClaims[i].Labels[v1.NodePoolLabelKey],
 		).V(1).Info("garbage collecting nodeclaim with no cloudprovider representation")
-		metrics.NodeClaimsTerminatedCounter.With(prometheus.Labels{
-			metrics.ReasonLabel:       "garbage_collected",
-			metrics.NodePoolLabel:     nodeClaims[i].Labels[v1.NodePoolLabelKey],
-			metrics.CapacityTypeLabel: metrics.GetLabelOrDefault(nodeClaims[i].Labels, v1.CapacityTypeLabelKey),
```

Comment:
Did we intentionally choose to drop the capacity label? Why are we dropping capacity now?

Reply:
Dropped it accidentally. Reverting it.